pkg/chrootarchive/diff_windows.go

```diff
@@ -14,6 +15,12 @@ import (
 // contents of the layer.
 func applyLayerHandler(dest string, layer archive.Reader, decompress bool) (size int64, err error) {
 	dest = filepath.Clean(dest)
+
+	// Ensure it is a Windows-style volume path
```

Comment:
For several older Windows APIs, if you provide a normal path like "C:\Foo" then you are limited to 260 characters.  Longer paths will produce errors, some with the clear "path too long" message but sometimes simply failing with confusing "path not found" messages.  To avoid this and enable long path handling, the path must be an absolute path prepended with "\\?\" like this code produces.